includes/class-newspack-newsletters-subscription.php

```diff
@@ -377,6 +377,7 @@ public static function add_contact( $contact, $lists = false ) {
 		$lists = apply_filters( 'newspack_newsletters_contact_lists', $lists, $contact, $provider->service );
 
 		$errors = new WP_Error();
+		$result = true;
```

Comment:
The returned value should either be an `array` or `WP_Error` and this variable should always have a value. Why create it early?

Reply:
Assigment of `$result` happens in a try/catch. If it errors out, the `$result` will never be assigned and so `is_wp_error( $result )` check will cause a PHP warning. I've updated the initial value to an empty array in d5339693d684c044c5943ce973da7f625eaff736